ocs_ci/ocs/ui/block_pool.py

```diff
@@ -173,3 +186,27 @@ def reach_pool_limit(self, replica, compression):
                     raise PoolStateIsUnknow(
                         f"pool {pool_name} is in unexpected state {pool_state}"
                     )
+
+    def pool_raw_capacity_loaded(self, pool_name):
```

Comment:
@shivamdurgbuns do update this with the format we follow.
eg: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/scale_lib.py#L101

Reply:
Have updated it @ramkiperiy